cpp/include/raft/core/logger-ext.hpp

```diff
@@ -24,6 +24,9 @@ namespace raft {
 static const std::string RAFT_NAME = "raft";
 static const std::string default_log_pattern("[%L] [%H:%M:%S.%f] %v");
 
+namespace detail {
+std::string format(const char* fmt, ...);
```

Comment:
I would consider adding `RAFT_INLINE_CONDITIONAL` here as well just for consistency with the function definition in `logger-inl.hpp`.

Reply:
Added.